kroxylicious-api/src/main/java/io/kroxylicious/proxy/filter/RequestFilter.java

```diff
@@ -49,6 +49,7 @@ default boolean shouldHandleRequest(ApiKeys apiKey, short apiVersion) {
      * @return a non-null CompletionStage that, when complete, will yield a RequestFilterResult containing the
      *         request to be forwarded.
      * @see io.kroxylicious.proxy.filter Creating Filter Result objects
+     * @see io.kroxylicious.proxy.filter Thread Safety
```

Comment:
Is the jvadoc tooling smart enough to link based on section titles or should this point at the actual id?

Reply:
I don't think it is currently possible.  I approach I've taken mirrored an approach I found in the JDK javadoc.  I can't remember where I saw it.

https://bugs.openjdk.org/browse/JDK-8301162 is coming, but that's not available until Java 21.

Reply:
Shame, at least jdk 21 is days away